docs/cli/operator.mdx

```diff
@@ -564,6 +571,367 @@ Use the options listed below to change the command's behavior.
+
+- [--output](/cli/cmd-options#output)
+
+- [--time-format](/cli/cmd-options#time-format)
+
+- [--tls](/cli/cmd-options#tls)
+
+- [--tls-ca-data](/cli/cmd-options#tls-ca-data)
+
+- [--tls-ca-path](/cli/cmd-options#tls-ca-path)
+
+- [--tls-cert-data](/cli/cmd-options#tls-cert-data)
+
+- [--tls-cert-path](/cli/cmd-options#tls-cert-path)
+
+- [--tls-disable-host-verification](/cli/cmd-options#tls-disable-host-verification)
+
+- [--tls-key-data](/cli/cmd-options#tls-key-data)
+
+- [--tls-key-path](/cli/cmd-options#tls-key-path)
+
+- [--tls-server-name](/cli/cmd-options#tls-server-name)
+
+#### update
+
+Update an existing Nexus Endpoint on the Server.
+
+An endpoint name is used in Workflow code to invoke Nexus operations. 
+The endpoint target may either be a Worker, in which case `--target-namespace` and
+`--target-task-queue` must both be provided, or an external URL, in which case
+`--target-url` must be provided.
+
+The endpoint is patched; existing fields for which flags are not provided are
+left as they were.
+
+Update only the target task queue:
+
+```
+temporal operator nexus endpoint update \
+  --name your-endpoint \
+  --target-task-queue your-other-queue
+```
+
+Update only the description:
+
+```
+temporal operator nexus endpoint update \
+  --name your-endpoint \
+  --description-file DESCRIPTION.md
+```
+
+Use the following options to change the behavior of this command.
+
+- [--address](/cli/cmd-options#address)
+
+- [--api-key](/cli/cmd-options#api-key)
+
+- [--codec-auth](/cli/cmd-options#codec-auth)
+
+- [--codec-endpoint](/cli/cmd-options#codec-endpoint)
+
+- [--color](/cli/cmd-options#color)
+
+- [--description](/cli/cmd-options#description)
+
+- [--description-file](/cli/cmd-options#description-file)
+
+- [--env](/cli/cmd-options#env)
+
+- [--env-file](/cli/cmd-options#env-file)
+
+- [--grpc-meta](/cli/cmd-options#grpc-meta)
+
+- [--log-format](/cli/cmd-options#log-format)
+
+- [--log-level](/cli/cmd-options#log-level)
+
+- [--name](/cli/cmd-options#name)
+
+- [--namespace](/cli/cmd-options#namespace)
+
+- [--no-json-shorthand-payloads](/cli/cmd-options#no-json-shorthand-payloads)
+
+- [--output](/cli/cmd-options#output)
+
+- [--target-namespace](/cli/cmd-options#target-namespace)
+
+- [--target-task-queue](/cli/cmd-options#target-task-queue)
+
+- [--target-url](/cli/cmd-options#target-url)
+
+- [--time-format](/cli/cmd-options#time-format)
+
+- [--tls](/cli/cmd-options#tls)
+
+- [--tls-ca-data](/cli/cmd-options#tls-ca-data)
+
+- [--tls-ca-path](/cli/cmd-options#tls-ca-path)
+
+- [--tls-cert-data](/cli/cmd-options#tls-cert-data)
+
+- [--tls-cert-path](/cli/cmd-options#tls-cert-path)
+
+- [--tls-disable-host-verification](/cli/cmd-options#tls-disable-host-verification)
+
+- [--tls-key-data](/cli/cmd-options#tls-key-data)
+
+- [--tls-key-path](/cli/cmd-options#tls-key-path)
+
+- [--tls-server-name](/cli/cmd-options#tls-server-name)
+
+- [--unset-description](/cli/cmd-options#unset-description)
+
```

Comment:
@fairlydurable Do you mind giving one quick look when you get back from PTO since the CLI is your realm of expertise? If there's anything major we can all probably hop on a call about it.